resources/views/components/molecules/_m-article-actions----digital-publication.blade.php

```diff
@@ -22,6 +22,36 @@
             @slot('currentArticle', $currentArticle ?? null)
         @endcomponent
     @endif
+
+    <ul class="m-article-actions{{ (isset($variation)) ? ' '.$variation : '' }}" aria-labelledby="h-article-actions{{ (isset($variation)) ? $variation : '' }}">
+
+        @if (empty($hideShare))
```

Comment:
I'm not sure that a slot value for `hideShare` and `pdfDownloadPath` are being passed to this molecule.

Reply:
I'm specifically looking at the call to the molecule in `digitalPublicationArticleDetail.blade.php`. In other places where `_m-article-actions.blade.php` is used to render the icons will appear, but in the case of digitalPublicationArticleDetail it calls `_m-article-actions----digital-publication.blade.php` directly, and should pass these values.